app/graphql/types/employees_type.rb

```diff
@@ -0,0 +1,15 @@
+class Types::EmployeeType < Types::BaseObject
+  field "id", ID
+  field "display_name", String
+  field "birthday", String
+  field "hire_date", String
+  field "is_photo_uploaded", String
+  field "photo_url", String
+end
+
+class Types::EmployeesType < Types::BaseObject
+  field "anniversary_employees", [Types::EmployeeType]
+  field "birthday_employees", [Types::EmployeeType]
+  field "time_off_employees", [Types::EmployeeType]
+  field "all_employees", [Types::EmployeeType]
+end
```

Comment:
This looks like it's redundant given the `employees(type: String)` query. It should probably be removed.

Reply:
This should be fixed now!